control-plane/agents/src/bin/ha/cluster/switchover.rs

```diff
@@ -647,6 +663,32 @@ impl From<Operation> for Stage {
     }
 }
 
+impl Event for SwitchOverRequest {
+    fn event(&self, action: EventAction) -> EventMessage {
+        let node_id = &self.node_name;
+        let status = match self.stage() {
+            Stage::Init => SwitchOverStatus::SwitchOverStarted,
+            Stage::Successful => SwitchOverStatus::SwitchOverCompleted,
+            Stage::Errored => SwitchOverStatus::SwitchOverFailed,
+            _ => SwitchOverStatus::UnknownSwitchOverStatus,
+        };
+
+        let event_source = EventSource::new(node_id.to_string()).with_switch_over_data(
+            status,
+            self.timestamp,
```

Comment:
Is this when the path down message is received from agent-ha-node?

Reply:
Yes it is.

No I was adding just the agent-ha-cluster events. What agent-ha-node events could be added?

Reply:
In Node agent there are things we could pick up on, path suspected, fail or "fixed". These would be very useful to have.

Reply:
I added nvme path suspect and fail events now.

Reply:
Missing one for when the path is "fixed" ?

Reply:
Can we get the fixed from node agent?

Reply:
Added NvmePathConnect event.